pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = [
     "typing_extensions>=0.0.0",
     "eval_type_backport>=0.0.0",
     "polars>=0.20.22",
+    "bibtexparser>=2.0.0b7",
```

Comment:
I would not add this here, it is only a requirement for the scripts, not the package)

Reply:
Done